Makefile

```diff
@@ -40,7 +40,7 @@ UNIT_TEST=$(shell go list ./... | grep -v bdd)
 
 test: generate fmt vet manifests
 	go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
-	setup-envtest use
+	setup-envtest --arch=amd64 use
```

Comment:
I do not think this m1 hack should be left alone on the repo, maybe make an extra Makefile target "test-m1" with this

Reply:
If `--arch=amd64` causes confusion in the `test` command, we can separate it. However, I do not believe that we really need to have 2 commands for the same purposes since amd64-based architectures are using `setup-envtest` with the `--arch=amd64` option by default https://github.com/kubernetes-sigs/controller-runtime/blob/b1d9a1641f312ed8b526506486415a3983d4e6d1/tools/setup-envtest/main.go#L43.